src/composables/biometricAuth.ts

```diff
@@ -0,0 +1,83 @@
+import { BiometricAuth } from '@aparajita/capacitor-biometric-auth';
+import { ref, readonly } from 'vue';
+
+import { tg as t } from '@/popup/plugins/i18n';
+import { IS_MOBILE_APP, MODAL_ENABLE_SECURE_LOGIN, MODAL_SECURE_LOGIN } from '@/constants';
+import { useUi } from './ui';
+import { useModals } from './modals';
+
+let composableInitialized = false;
+const isAvailable = ref(false);
+const isAuthenticated = ref(false);
+
+export function useBiometricAuth() {
```

Comment:
What do you think about making it more generic by not focusing only on the biometric aspect? This way we could extend this composable with different methods of athenticating users.
```suggestion
export function useAuth() {
```